### PR TITLE
Rutefig/reg 587 automatically set max length is too short for extracted

### PR DESCRIPTION
**Pull request by rutefig**
## Summary

This PR fixes issues with regex pattern validation in the blueprint creation flow, ensuring proper error display and preventing blueprint submission when there are validation errors. It also corrects the SDK package reference in the bun lock file.

## Problem

The blueprint creation interface had multiple issues that created a poor user experience:
1. Users could submit blueprints even when regex patterns had validation errors
2. Error states for regex patterns weren't being properly displayed
3. The bun lock file was referencing a local SDK package path instead of the published npm package
4. The validation logic for determining if compilation should be allowed was inconsistent

## Solution

Improved the validation logic in the ExtractFields component to properly handle error states and prevent submission when errors exist:
- Added explicit error state checking using `regexGeneratedOutputErrors.length > 0`
- Updated the compile button to be disabled when there are any errors present (`errors.length > 0`)
- Fixed the output display condition to show outputs regardless of error state
- Corrected the SDK package reference in bun.lock to use the npm registry version

## Changes

- Fixed bun lock file to reference `@zk-email/sdk@1.0.3` from npm instead of local file path
- Enhanced error validation logic in ExtractFields component to properly detect and display regex pattern errors
- Added compile button disabling when validation errors exist
- Improved output field visibility logic to always show when regex parts or outputs exist
- Added console logging for debugging regex outputs (to be removed if not needed in production)

### Automated Tests
- [x] Unit tests pass (no unit test suite configured)
- [ ] Integration tests pass (not applicable)
- [ ] E2E tests pass (not run - requires playwright setup)

### Manual Testing
- [ ] Tested locally with development environment
- [ ] Tested edge cases
- [ ] Verified no regressions

## Breaking Changes

- None

## Checklist

- [x] Code follows the project's style guidelines
- [x] Self-review completed
- [ ] Comments added for complex code sections
- [ ] Documentation updated if needed
- [x] No new warnings generated (build passes with existing warnings)
- [ ] Tests added/updated to cover changes
- [x] All tests passing locally (build passes successfully)

## Related Issues

- Closes #587 (Automatically set max length is too short for extracted fields)

## Additional Notes

The main improvements focus on the user experience during blueprint creation by providing clearer error states and preventing invalid blueprints from being compiled. The console.log statement on line 154 of ExtractFields.tsx should be removed before merging if not needed for production debugging.

## Summary by CodeRabbit

* **Bug Fixes**
  * Stricter validation: prompts now appear only when there are truly no outputs and no output errors.
  * Compile button now disables when validation errors exist.
  * Output area now renders whenever parsed parts, generated outputs, or any errors are present and correctly shows errors or combined output.
  * Removed extraneous console logging.

* **Chores**
  * Added new ignore patterns to the repository ignore list.

### CODE REVIEW

File: src/app/create/[id]/createBlueprintSteps/ExtractFields.tsx [hunk 1]
```diff
@@ -304,9 +305,7 @@ const ExtractFields = ({
     }
     if (
       !regexGeneratedOutputs.length ||
-      regexGeneratedOutputs.some((output) =>
-        Array.isArray(output) ? output.join('').includes('Error') : output.includes('Error')
-      )
+      regexGeneratedOutputErrors.length > 0
     ) {
```
[the remark this answers is not in the document]
Comment (rutefig) [reply src/app/create/[id]/createBlueprintSteps/ExtractFields.tsx]: fixed

File: src/app/create/[id]/createBlueprintSteps/ExtractFields.tsx [hunk 2]
```diff
@@ -837,10 +834,9 @@ const ExtractFields = ({
                   </Button>
                 </div>
                 {/* Only show output if there are regex parts and valid outputs */}
-                {parseRegexParts(regex.parts).length > 0 &&
-                regexGeneratedOutputs[index] !== undefined &&
-                regexGeneratedOutputs[index] !== null &&
-                regexGeneratedOutputs[index].length > 0 ? (
+                {
+                parseRegexParts(regex.parts).length > 0 ||
+                regexGeneratedOutputs.length > 0 ? (
```
Comment (wryonik) [inline src/app/create/[id]/createBlueprintSteps/ExtractFields.tsx]: Here the individual elems can be undefined and null and that will still be valid as we are only checking the length. Why are we doing this?
